### DISCUSSION TITLE
Email Notifcations

### DISCUSSION THREAD

**Discussion by BillingServ** [towfiqi/serpbear#38] [General]
Hey, 

Is there any way to test notifications? Not currently getting them, and can't see anything in the log about them either; it looks like the cron completes then doesn't go onto the notification, which is set to daily. it's been set up for a good few days now and not got one email. I could be blind, but everything is setup correctly as far as I can see. 

Thanks,
Jord

**Comment by towfiqi** [comment]
Jord, will add better error logging in the next release. For now, can you please check your log in your SMTP service provider?

**Reply by BillingServ** [reply]
Thank you for that; I've checked our SMTP settings everything is correct; I've also checked our SMTP logs, and it doesn't look like it's even trying to send anything.

**Reply by BillingServ** [reply]
Using Docker too, if that helps.

**Reply by towfiqi** [reply]
Its probably becuase the CRON time was wrong. Instead of running the email notification cron everyday at 3am, it was actually set to run 3rd day of every month. Released a new version with lots of new features and bug fixes. Please pull the latest docker image and it should fix the issue.

**Reply by BillingServ** [reply]
That's great thank you, I've pulled the latest image and will test it out, thank you again for getting that fixed so quickly.

**Reply by BillingServ** [reply]
All works now, I got an email this morning. Thanks for getting this fixed.

**Reply by towfiqi** [reply]
Great!
